src/runtime.cc

```diff
@@ -149,7 +149,7 @@ namespace detail {
 		m_reduction_mngr = std::make_unique<reduction_manager>();
 		m_host_object_mngr = std::make_unique<host_object_manager>();
 
-		if(m_cfg->is_recording()) m_task_recorder = std::make_unique<task_recorder>(m_buffer_mngr.get());
+		if(m_cfg->should_print_graphs()) m_task_recorder = std::make_unique<task_recorder>(m_buffer_mngr.get());
```

Comment:
I'd prefer if this was still controlled by a configuration specifically referring to recording in terms of naming (especially in light of soon-upcoming future uses of recording).

Reply:
I've added `config::should_record()` as (for now) an alias to `should_print_graphs()`.